src/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.py

```diff
@@ -191,7 +191,7 @@ def _get_regex_callbacks(self) -> list[RegexCallback]:
 
             completed_regexes = [re.compile(".*Finished Rendering.*")]
             progress_regexes = [re.compile(".*Progress ([0-9]+)%.*")]
-            error_regexes = [re.compile(".*Error: .*|.*\\[Error\\].*", re.IGNORECASE)]
+            error_regexes = [re.compile(".*Error: .*|.*\\[Error\\].*|.*CRITICAL: .*", re.IGNORECASE)]
```

Comment:
Can we add unit tests to confirm that any CRITICAL / ERROR in the log stops the adaptor?

Reply:
Thanks for adding the test for Critical errors.